osu.Game/Users/UserPanel.cs

```diff
@@ -61,27 +61,31 @@ protected UserPanel(APIUser user)
         [Resolved]
         protected OsuColour Colours { get; private set; } = null!;
 
+        protected virtual bool ShouldCreateUserCoverBackground { get; set; } = true;
```

Comment:
I have a real strained relationship with these sorts of properties. Half the time these work ok, half the time they turn out to be a horrible mistake N months later down the lane when another design comes that doesn't fit the overfitted abstraction that these provide.

It's probably fine to have it for now, but it's written weird/wrong right now. It should be either a `get`-only virtual property, or a `get; init` non-virtual one - having it settable at any point is kind of a lie since it only has an effect when read once in BDL.

If I *were* to change this any further, I'd probably go for something like a `IEnumerable<Drawable> CreateBackground()` method, that in this implementation creates the box and the cover, and only the box in the one that doesn't want the cover. That's *slightly* less specific than this current setup.

Reply:
Sure, I'll go with the `CreateBackground()`